packages/graphql/lib/getDefaultResolvers/MutationBuilder/resolveCreateMutation.ts

```diff
@@ -2,16 +2,21 @@ import { IMutationViewMeta } from "@fullstack-one/schema-builder";
 import { IParsedResolveInfo } from "../types";
 import { IMutationBuildObject, IMutationInputObject } from "./types";
 import parseValue from "./parseValue";
+import { ReturnIdHandler } from "../../ReturnIdHandler";
 
-export default function resolveCreateMutation(query: IParsedResolveInfo<IMutationInputObject>, mutation: IMutationViewMeta): IMutationBuildObject {
+export default function resolveCreateMutation(
+  query: IParsedResolveInfo<IMutationInputObject>,
+  mutation: IMutationViewMeta,
+  returnIdHandler: ReturnIdHandler
+): IMutationBuildObject {
   const fieldNames = Object.keys(query.args.input)
     .map((name) => `"${name}"`)
     .join(", ");
 
   const values: string[] = [];
   const valuesString: string = Object.values(query.args.input)
     .map((value, index) => {
-      values.push(parseValue(value));
+      values.push(parseValue(value, returnIdHandler));
```

Comment:
> `values.push(parseValue(value, returnIdHandler));`

Did you mean:
`values.push(parseValue(value, returnIdHandler.getReturnId(???)));`